runtime/__init__.py

```diff
@@ -125,6 +125,21 @@ def load_method(self, name: str) -> Optional[Method]:
         return self._methods.get(name, None)
 
 
+class BackendRegistry:
+    """The registry of backends that are available to the runtime."""
+
+    def __init__(self, legacy_module: ModuleType) -> None:
+        # TODO: Expose the kernel callables to Python.
+        self._legacy_module = legacy_module
+
+    @property
+    def _get_backend_names(self) -> Set[str]:
+        """
+        Returns the names of all registered backends as a list of strings.
```

Comment:
as a set?

Reply:
list is better. Borrowed some code from `get_operators_names` but now I don't think we will have duplication